CMake Error at layers/CMakeLists.txt:318 (target_link_libraries):
  Target "vvl" links to:

    SPIRV-Headers::SPIRV-Headers

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

--- CMakeLists.txt.orig	2023-12-20 00:20:05 UTC
+++ CMakeLists.txt
@@ -79,6 +79,9 @@ find_package(SPIRV-Headers CONFIG QUIET)
 find_package(VulkanUtilityLibraries CONFIG QUIET)
 
 find_package(SPIRV-Headers CONFIG QUIET)
+if (NOT TARGET SPIRV-Headers::SPIRV-Headers)
+    add_subdirectory(SPIRV-Headers EXCLUDE_FROM_ALL SYSTEM)
+endif()
 
 find_package(SPIRV-Tools-opt CONFIG QUIET)
 
